src/components/PlansTab/PlansTab.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { bgImageStyle } from 'util/index'
+import { capitalize } from 'lodash'
+import { useCurrentUser } from 'hooks/useCurrentUser'
+import { useEnsureGroupPlans } from 'hooks/useEnsureGroupPlans'
+import { useEnsureUserPlans } from 'hooks/useEnsureUserPlans'
+import Pill from 'components/Pill'
+import Icon from 'components/Icon'
+import RoundImage from 'components/RoundImage'
+import {
+  DEFAULT_BANNER,
+  DEFAULT_AVATAR
+} from 'store/models/Group'
+
+import './plansTab.scss'
+
+export default function PlansTab ({ group = {} }) {
+  const currentUser = useCurrentUser()
+  const isGroupView = !!group.id
+  const { moderatorPlans, userPlans } = useEnsureUserPlans(currentUser.id)
+  const groupPlans = useEnsureGroupPlans(group.id)
+
+  /*
+    What data is needed?
+    - group from props
+    - currentUser
+    - (groupTab) all plans associated with a group
+    - (userTab) all plans for groups the user moderates
+    - (userTab) all plans a user is subscribed to
+
+    What needs to be shown?
+    group tab
+    - A 'Create plan' button to navigate to the plan creation
+    - list of plans associated with group
+      - Highlight any plans the currentUser is linked to
```

Comment:
can a user be part of more than one plan in a group? is there a use case for this?

Reply:
No, I don't think so.